vividus-engine/src/main/java/org/vividus/context/RunContext.java

```diff
@@ -34,7 +34,9 @@
 
     boolean isDryRun();
 
-    boolean isRunCompleted();
-
     void completeRun();
+
+    void startRun();
+
+    RunState getRunState();
```

Comment:
```suggestion
    void startRun();
    
    boolean isRunInProgress();
    
    void completeRun();
```

and then make enum private